tensorflow/compiler/xla/service/cpu/BUILD

```diff
@@ -295,6 +295,7 @@ cc_library(
         "@llvm//:object",
         "@llvm//:support",
         "@llvm//:target",
+        "@llvm//:polly",
```

Comment:
To add a conditional take a look at how if_mkl works as an example.  Some links to help you get started:

 - Definition for if_mkl: https://github.com/tensorflow/tensorflow/blob/master/third_party/mkl/build_defs.bzl
 - The corresponding config_setting: https://github.com/tensorflow/tensorflow/blob/master/third_party/mkl/BUILD
 - Documentation for config_setting: https://docs.bazel.build/versions/master/be/general.html#config_setting

However, before you go ahead and do a bunch of work, I think we should take a step back and reach agreement on whether Polly constitutes a supported configuration.  By "supported configuration" I mean that it is linked into TensorFlow by default and breaking a Polly test case is sufficient cause to revert a patch or to hold a push.  My perspective on this is that it isn't fair to ask Google and other TF developers to pay the cost of supporting Polly till we've shown a clear benefit (better runtime performance, basically) from Polly; and we should start off by designating the TF-with-Polly build as "experimental" -- it won't be built by default and the onus of keeping it working will fall on you.  Once we are able to demonstrate value from Polly, and have community support to address breakages and regressions in Polly, we can discuss enabling it by default as a supported configuration.